x-pack/plugins/observability_solution/observability_ai_assistant_app/public/functions/visualize_esql.tsx

```diff
@@ -259,150 +259,156 @@ export function VisualizeESQL({
-                            {
-                              defaultMessage: 'Display chart',
-                            }
-                          )
-                        : i18n.translate(
-                            'xpack.observabilityAiAssistant.lensESQLFunction.displayTable',
+                            'xpack.observabilityAiAssistant.lensESQLFunction.visualization',
                             {
-                              defaultMessage: 'Display results',
+                              defaultMessage: 'Visualization',
                             }
                           )
+                        : i18n.translate('xpack.observabilityAiAssistant.lensESQLFunction.table', {
+                            defaultMessage: 'Table of results',
+                          })
                     }
-                  />
-                </EuiToolTip>
-              </EuiFlexItem>
-              <EuiToolTip
-                content={i18n.translate('xpack.observabilityAiAssistant.lensESQLFunction.edit', {
-                  defaultMessage: 'Edit visualization',
-                })}
-              >
-                <EuiButtonIcon
-                  size="xs"
-                  iconType="pencil"
-                  onClick={() => {
-                    chatFlyoutSecondSlotHandler?.setVisibility?.(true);
-                    if (triggerOptions) {
-                      uiActions.getTrigger('IN_APP_EMBEDDABLE_EDIT_TRIGGER').exec(triggerOptions);
-                    }
-                  }}
-                  data-test-subj="observabilityAiAssistantLensESQLEditButton"
-                  aria-label={i18n.translate(
-                    'xpack.observabilityAiAssistant.lensESQLFunction.edit',
-                    {
-                      defaultMessage: 'Edit visualization',
-                    }
-                  )}
-                />
-              </EuiToolTip>
-              <EuiFlexItem grow={false}>
+                  >
+                    <EuiButtonIcon
+                      size="xs"
+                      iconType={isTableVisible ? 'visBarVerticalStacked' : 'tableDensityExpanded'}
+                      onClick={() => setIsTableVisible(!isTableVisible)}
+                      data-test-subj="observabilityAiAssistantLensESQLDisplayTableButton"
+                      aria-label={
+                        isTableVisible
+                          ? i18n.translate(
+                              'xpack.observabilityAiAssistant.lensESQLFunction.displayChart',
+                              {
+                                defaultMessage: 'Display chart',
+                              }
+                            )
+                          : i18n.translate(
+                              'xpack.observabilityAiAssistant.lensESQLFunction.displayTable',
+                              {
+                                defaultMessage: 'Display results',
```

Comment:
Should we change this to "Display table" as well?

Reply:
Done https://github.com/elastic/kibana/pull/184923/commits/58f2c3832485dad9f5d25ca044aeef8bfdf5bac8